hypogeum/src/test/kotlin/io/github/campaignery/hypogeum/data/ContestantDataSourceTest.kt

```diff
@@ -0,0 +1,26 @@
+package io.github.campaignery.hypogeum.data
+
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class ContestantDataSourceTest {
+
+    private val dataSource: ContestantDataSource = ContestantDataSource
+
+    @Test
+    fun `Verify warrior doesn't exceed available points`() {
+        for (warrior in dataSource.contestants()) {
+            assertTrue(warrior.isValid)
```

Comment:
```suggestion
    fun `Verify contestant doesn't exceed available points`() {
        for (contestant in dataSource.contestants()) {
            assertTrue(contestant.isValid)
```
Since contestants may be something other than a warrior in the future 😅

Reply:
Good catch 🎣 yeah, I did some refactoring and missed this one 🙈